src/components/molecules/HeatMap.tsx

```diff
@@ -32,11 +32,13 @@ const HeatMap: FC<IProps> = ({ data, center, sizeOptions }) => {
 
   const mapRef = React.createRef<any>();
   useEffect(() => {
-    const map = mapRef.current.leafletElement;
-    setTimeout(() => {
-      map.invalidateSize();
-    });
-  }, [sizeOptions, mapRef]);
+    if(mapRef.current) {
+      const map = mapRef.current.leafletElement;
+      setTimeout(() => {
```

Comment:
add a comment here with reference to https://github.com/OpenGov/react-leaflet-heatmap-layer/issues/49

Reply:
@yuvalbl  [this site](https://github.com/OpenGov/react-leaflet-heatmap-layer/issues/49) show 404 error.
maybe they delete the issues..